packages/ERTP/src/paymentLedger.js

```diff
@@ -14,6 +14,11 @@ import { BrandI, makeIssuerInterfaces } from './typeGuards.js';
 
 const { details: X, quote: q, Fail } = assert;
 
+/**
+ * @param {Brand} brand
+ * @param {AssetKind} assetKind
+ * @param {Matcher} elementShape
```

Comment:
```suggestion
 * @param {Pattern} elementShape
```

For example, `harden({ x: M.bigint() })` is a `Pattern` but not a `Matcher`, whereas `M.bigint()` is both a `Pattern` and a `Matcher`.